src/pages/jobs/desktop/desktop.tsx

```diff
@@ -1,35 +1,41 @@
+import { useNavigate } from '@tanstack/react-location';
 import { TwoColumnCursor } from 'src/components/templates/two-column-cursor/two-column-cursor';
-import css from './desktop.module.scss';
-import { Card } from 'src/components/atoms/card/card';
-import { Avatar } from 'src/components/atoms/avatar/avatar';
+import { ProfileCard } from 'src/components/templates/profile-card';
 import { CardMenu } from 'src/components/molecules/card-menu/card-menu';
 import { JobList } from 'src/components/organisms/job-list/job-list';
-import { NetworkMenuList } from '../jobs.services';
-import { useJobsShared } from '../jobs.shared';
 import { printWhen } from 'src/core/utils';
+import { useJobsShared } from '../jobs.shared';
+import css from './desktop.module.scss';
 
 export const Desktop = (): JSX.Element => {
-  const { onMorePage, jobList, avatarImg, name, navigateToProfile, jobsMenuListUser, jobsMenuListOrg, identity } =
-    useJobsShared();
+  const navigate = useNavigate();
+  const { onMorePage, jobList, identity } = useJobsShared();
+
+  const NetworkMenuList = [
+    { label: 'Connections', icon: '/icons/network.svg', link: () => navigate({ to: '/network/connections' }) },
+    { label: 'Followings', icon: '/icons/followers.svg', link: () => navigate({ to: '/network/followings' }) },
+  ];
+
+  const jobsMenuListUser = [
+    {
+      label: 'My applications',
+      icon: '/icons/my-applications.svg',
+      link: () => navigate({ to: `/jobs/applied/${identity.id}` }),
```

Comment:
the `jobs/applied` and `jobs/created` routes had some changes. I will update them. please add a todo here in this line.

Reply:
ok